docs/docsite/rst/user_guide/sample_setup.rst

```diff
@@ -37,29 +37,9 @@ This layout organizes most tasks in roles, with a single inventory file for each
         webservers-extra.yml  # <-- avoids confusing playbook with task files
 
     roles/
-        common/               # this hierarchy represents a "role"
-            tasks/            #
-                main.yml      #  <-- tasks file can include smaller files if warranted
-            handlers/         #
-                main.yml      #  <-- handlers file
-            templates/        #  <-- files for use with the template resource
-                ntp.conf.j2   #  <------- templates end in .j2
-            files/            #
-                bar.txt       #  <-- files for use with the copy resource
-                foo.sh        #  <-- script files for use with the script resource
-            vars/             #
-                main.yml      #  <-- variables associated with this role
-            defaults/         #
-                main.yml      #  <-- default lower priority variables for this role
-            meta/             #
-                main.yml      #  <-- role dependencies
-            library/          # roles can also include custom modules
-            module_utils/     # roles can also include custom module_utils
-            lookup_plugins/   # or other types of plugins, like lookup in this case
-
-        webtier/              # same kind of structure as "common" was above, done for the webtier role
-        monitoring/           # ""
-        fooapp/               # ""
+        role1/                # See the role documentation for explaination of a role directory structrure 
```

Comment:
While this is potentially good to deduplicate examples, I'm never going to find the docs without a direct link.  We should find a way to reference an actual URL to go see the examples.

Maybe add a note below so we can use a link.  I'll ask for ideas from the docs team

Reply:
I  think the idea is valid - try not to have too much duplicated content, especially as it can get updated in one location and we forget about a second location. That said, this information is useful imo for what this page is trying to do - give a full example of the layout of an Ansible project.  I'll try some local rst experiments to see if there is a way to use a snippet here and just include it in both places, but my feeling is that won't work since it's inside a code example so to speak.

@jdorel - can you explain why you wanted to remove the roles details here? Is there something we're missing that led to this PR?  Thanks.

Reply:
Aside from the duplication of roles details, the main reason I would like role details to be removed from this page is because it makes the pages harder to read. When you first discover ansible and you see this sample setup, you think : `wow, this looks so complex`.
Looking at you experiment below, maybe it could be interesting to include the details of a rule in a subsection ?